sds_data_manager/lambda_code/SDSCode/database/database.py

```diff
@@ -1,5 +1,32 @@
-"""Create engine that will be used to create schema or make query"""
-from SDSCode.database import config
+"""Create database URI that will be used to create engine or make query"""
+
+import json
+import os
+
+import boto3
 from sqlalchemy import create_engine
 
-engine = create_engine(config.db_uri, echo=True)
+
+def get_db_uri():
```

Comment:
Can this live in `get_engine()` now? Just put a comment saying you first need to get the DB uri and add the content from here down below in that function.